api/openapi-spec/swagger.json

```diff
@@ -86423,6 +86423,9 @@
    },
    "io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinition": {
     "description": "CustomResourceDefinition represents a resource that should be exposed on the API server.  Its name MUST be in the format \u003c.spec.name\u003e.\u003c.spec.group\u003e.",
+    "required": [
+     "spec"
```

Comment:
I actually expected changes in the openapi spec because of the `// +optional` changes but the openapi spec seems to be generated using the omitempty tag...

@mbohlool am I looking at this the wrong way? Are the `// +optional` tags used for generating the openapi spec? :thinking: